openjdk/ProblemList_openjdk11-openj9.txt

```diff
@@ -46,9 +46,9 @@ java/lang/StackTraceElement/SerialTest.java	https://github.com/eclipse/openj9/is
 java/lang/StackWalker/Basic.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
 java/lang/StackWalker/DumpStackTest.java	https://github.com/eclipse/openj9/issues/6680	generic-all
 java/lang/StackWalker/EmbeddedStackWalkTest.java	https://github.com/eclipse/openj9/issues/3941	generic-all
-java/lang/StackWalker/LocalsAndOperands.java#id0  https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297	generic-all
-java/lang/StackWalker/LocalsAndOperands.java#id1  https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297	generic-all
-java/lang/StackWalker/ReflectionFrames.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297	generic-all
+java/lang/StackWalker/LocalsAndOperands.java#id0  https://github.com/AdoptOpenJDK/openjdk-tests/issues/1282	generic-all
+java/lang/StackWalker/LocalsAndOperands.java#id1  https://github.com/AdoptOpenJDK/openjdk-tests/issues/1282	generic-all
+java/lang/StackWalker/ReflectionFrames.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1284	generic-all
 java/lang/StackWalker/VerifyStackTrace.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1273	generic-all
```

Comment:
These were originally under 1297, as permanent excludes.  What is the purpose of changing them to 1282/1284 issues?

Reply:
The original issues provide the reasoning behind the permanent exclusion or in case these tests are patched for openj9 and can be reopened